src/pages/Embeds/Plio.vue

```diff
@@ -399,49 +400,32 @@ export default {
   methods: {
     ...mapActions("auth", ["setAccessToken", "setActiveWorkspace"]),
     /**
-     * Update the number of correct answers
-     * @param  {Number} itemIndex -  The index of the item to be considered for the calculation
-     * @param  {String, Number, Object} userAnswer - User's answer to that item
+     * Checks whether the item at the provided itemIndex is a subjective question
+     * and if the user has answered the question or not
+     * @param {Number} itemIndex - The index of the item in question
```

Comment:
writing `in question` here can be confusing :p

```suggestion
     * @param {Number} itemIndex - The index of the item to be checked
```